app/assets/stylesheets/general/_titles.scss

```diff
@@ -47,6 +47,10 @@ h6 {
   font-size: $font-size-base;
 }
 
+:is(h1, h2, h3, h4, h5, h6) .header_anchor {
```

Comment:
Unfortunately this isn't supported by safari 13 which is still used by a significant enough portion of users to need support for it. But luckily we are using SCSS so instead you can nest it like

```
h1, h2, h3... {
   .header-anchor
}
```

Perhaps just targeting the link tag instead of giving it an explicit class name would be enough too `a[href^="#"]`, although it doesn't matter too much